ui/app.js

```diff
@@ -3778,149 +3778,166 @@ $(document).ready(() => {
-        $(".manage-profile-name-input-1").attr("readonly", true);
-        $(".manage-profile-name-input-2").attr("readonly", true);
-        $("#reports-officers-involved-tag-title").html(
-          "Officers Involved"
-        );
-        $(".roster-iframe").attr("src", rosterLink);
-
-        $(".manage-profile-save").css("display", "block");
-        $(".manage-profile-editing-title").css("display", "block");
-        $(".manage-incidents-create").css("display", "block");
-        $(".manage-incidents-save").css("display", "block");
-        $(".manage-incidents-editing-title").css("display", "block");
-        $(".manage-reports-new").css("display", "block");
-        $(".manage-reports-save").css("display", "block");
-        $(".manage-reports-editing-title").css("display", "block");
-        $(".vehicle-information-save").css("display", "block");
-        $(".vehicle-information-title").css("margin-right", "0px").css("width", "81%");
-        $(".manage-incidents-title ").css("margin-right", "0px")
-        $(".manage-reports-title").css("margin-right", "0px").css("width", "66%");
-        } else if (sentJob == "sasp")  {
-          document.documentElement.style.setProperty(
-            "--color-1", /* Panels */
-            "#423f39"
-          );
-          document.documentElement.style.setProperty(
-            "--color-2", /* Clock */
-            "#8f7c3f"
-          );
-          document.documentElement.style.setProperty(
-            "--color-3", /* Outlines and hover */
-            "#16537e"
-          );
-          document.documentElement.style.setProperty(
-            "--color-4", /* Button Base */
-            "#8f7c3f"
-          );
-          document.documentElement.style.setProperty(
-            "--color-5", /* interactables */
-            "#0f3a58"
-          );
-          document.documentElement.style.setProperty(
-            "--color-6", /* context text box? */
-            "#121f2c"
-          );
-          document.documentElement.style.setProperty(
-            "--color-7", /* Dispatch Msg box */
-            "#0f3a58"
-          );
-          document.documentElement.style.setProperty(
-            "--color-8", /* ? */ /* ? */
-            "#2554cc"
-          );
-          document.documentElement.style.setProperty(
-            "--color-9", /* backround */
-            "#9c9485"
-          );
-          document.documentElement.style.setProperty(
-            "--color-10", /* Refresh button */
-            "#8F741B"
-        );
-        $(".badge-logo").attr("src", "img/sasp_badge.webp");
-        $(".header-title").html("SAN ANDREAS STATE POLICE");
+  }
+  function JobColors(sentJob) {
+    if (sentJob) {
+      if (PoliceJobs[sentJob] !== undefined)  {
+        if (sentJob == "police") {
+            applyCustomTheme(customThemes.lspd)
+            $(".badge-logo").attr("src", "img/LSPD.webp");
+            $(".header-title").html("LOS SANTOS POLICE");

```

Comment:
Thanks for applying these changes, it looks so much cleaner!

If you want you could even add the `img` and the `title` as variables on your customThemes config. Then you only need to call `applyCustomTheme`.

supernit: Feel free to ignore, but there are two existing sections that were applying custom styles for the EMS and DOJ themes, they would benefit from this refactoring work you've done as well, if you're feeling up for cleaning up a bit more code :P Look for `AmbulanceJobs[sentJob]` and `DojJobs[sentJob]`.

Reply:
yeah, I'm learning JS as I go so I appreciate the help. I noticed that I could apply the same process to the other jobs as i did the police but I decided to push the police changes because they were functional then work on other things before i break something. But do plan on re doing the other jobs the same way i did police because better to do it now before its requested by someone.